src/core/PortalManager.cpp

```diff
@@ -19,13 +19,19 @@ SOutput::SOutput(SP<CCWlOutput> output_) : output(output_) {
 
         Debug::log(LOG, "Found output name {}", name);
     });
-    output->setMode([this](CCWlOutput* r, uint32_t flags, int32_t width, int32_t height, int32_t refresh) { //
-        refreshRate = refresh;
-    });
-    output->setGeometry([this](CCWlOutput* r, int32_t x, int32_t y, int32_t physical_width, int32_t physical_height, int32_t subpixel, const char* make, const char* model,
-                               int32_t transform_) { //
-        transform = (wl_output_transform)transform_;
+    output->setMode([this](CCWlOutput* r, uint32_t flags, int32_t width, int32_t height, int32_t refresh) {
+        refreshRate  = refresh;
+        this->width  = width;
```

Comment:
no this-> here and everywhere else

for conflicting local and member names, use `local_` e.g. `int32_t width_` and then `width = width_`